test/test-clib-override.cc

```diff
@@ -53,36 +53,35 @@ TEST_F(CLIBOverrides, open) {
     EXPECT_EQ(__getfdccsid(fd), 0x10000 + 819);
     close(fd);
 
+    char buff[] = "This is a test";
+    int bufflen = strlen(buff);
```

Comment:
Couple issues:
1. that should be strlen(buff)+1
2. use sizeof(buff) as that is compile time value
3. make this constexpr or leave the code using sizeof(buff)